sdk/identity/Azure.Identity/tests/EnvironmentCredentialProviderTests.cs

```diff
@@ -113,52 +104,26 @@ public async Task EnvironmentCredentialAuthenticationFailedException()
             await Task.CompletedTask;
         }
 
-        [NonParallelizable]
-        [Test]
-        public void AssertCredentialUnavailableWhenEmptyString()
+        public static IEnumerable<object[]> EnvironmentSettings()
```

Comment:
Should this have a more specific name since it's used to drive a specific test? Perhaps something like `AssertCredentialUnavailableWhenEmptyStringEnvironmentSettings` (though perhaps that's too verbose).

Reply:
Descriptive is good with me :)